README.md

```diff
@@ -121,4 +109,17 @@ You can then start irmago with the store-type flag set to Redis and the [default
 irma server -vv --store-type redis --redis-addr "localhost:6379" --redis-allow-empty-password
 ```
 
+
+### Running without Go
+
+You can also run the tests fully in Docker using the command below. This is useful when you don't want to install the Go compiler locally. By default, all tests are run one-by-one without parallel execution.
+
+    docker-compose run test
+
+You can override the default command by specifying command line options for `go test` manually, for example:
+
+    docker-compose run test ./internal/sessiontest -run TestDisclosureSession
+
+We always enforce the `-p 1` option to be used (as explained [above](#running-the-tests)).
+
```

Comment:
I think something went wrong while merging this file, because the 'Running without Go' instructions are about the unit tests and now they have become part of the Redis instructions.